core/tests/puppeteer-acceptance-tests/blog-admin-tests/blog-admin-assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Accpetance Test for a Blog Admin to assign roles and change
+ * tag properties from the blog-admin page.
+ */
+
+const createNewUser = require(
+  '../puppeteer-testing-utilities/initializeUsers.js');
+
+const ROLE_BLOG_ADMIN = 'blog admin';
+const ROLE_BLOG_POST_EDITOR = 'blog post editor';
+
+let blogAdminAssignRolesToUsersAndChangeTagProperties = async function() {
+  const superAdmin = await createNewUser.superAdmin(
+    'superAdm', 'testadmin@example.com', ROLE_BLOG_ADMIN);
+  await createNewUser.blogAdmin(
+    'blogAdm', 'blog_admin@example.com');
+  await createNewUser.blogPostEditor(
+    'blogEditor', 'blog_post_editor@example.com');
+
+  await superAdmin.assignUserAsRoleFromBlogAdminPage('blogAdm', 'BLOG_ADMIN');
+  await superAdmin.expectUserToHaveRole('blogAdm', ROLE_BLOG_ADMIN);
+  await superAdmin.assignUserAsRoleFromBlogAdminPage(
```

Comment:
Isn't the blog admin (not the super admin) the one who's supposed to be assigning the blog editor role -- isn't that what we're supposed to be testing? Ditto for the rest of the tests.